include/IECoreHoudini/LiveScene.h

```diff
@@ -59,7 +59,8 @@ class LiveScene : public IECore::SceneInterface
 		
 		LiveScene();
 		LiveScene( const UT_String &nodePath, const Path &contentPath, const Path &rootPath, double defaultTime = std::numeric_limits<double>::infinity() );
-		
+		LiveScene( const UT_String &nodePath, const Path &contentPath, const Path &rootPath, double defaultTime, DetailSplitter *splitter );
```

Comment:
I can't remember why exactly, but I think it was important the the public constructor had the default value for `defaultTime` set to `std::numeric_limits<double>::infinity()`